configure.ac

```diff
@@ -273,7 +273,16 @@ if test "$have_tiff" = yes; then
     GRAPHIC_LFLAGS="$with_tiff_library $GRAPHIC_LFLAGS"
     GRAPHIC_CFLAGS="$with_tiff_include $GRAPHIC_CFLAGS"
   else
-    have_tiff=no
+    PKG_CHECK_MODULES(
+      [TIFF],
+      [libtiff-4],
+      [
+        GRAPHIC_CFLAGS="$TIFF_CFLAGS $GRAPHIC_CFLAGS"
```

Comment:
Why did you switch this two lines?

Reply:
I copy/pasted the code which used `PKG_CHECK_MODULES` to search for ImageMagick, and that had `LDFLAGS` and `CFLAGS` in this order.  Good catch; I amended the commit to keep the order consistent with the previous lines.